pkg/drivers/hyperv/hyperv_windows.go

```diff
@@ -303,6 +305,11 @@ func (d *Driver) Stop() error {
 
 // Remove removes an host
 func (d *Driver) Remove() error {
+	if _, stderr, err := powershell.Execute(`Hyper-V\Get-VM`, d.MachineName); err != nil &&
+		strings.Contains(stderr, "Hyper-V was unable to find a virtual machine") {
```

Comment:
are calls forced to the English locale?

Reply:
```
get-vm -name crc -ErrorAction SilentlyContinue 
```

would error when the VM does not exist, and else provide the info... in that case the return code can be checked for

Reply:
> are calls forced to the English locale?

No

> -ErrorAction SilentlyContinue 

I don't understand how it works. What do you suggest exactly?

Reply:
if not using an English locale, checking "Hyper-V was unable to find a virtual machine" might not give expected results. the a;ternative is to check only the return code with slientlycontinue alone. but might not be a necessity.

Reply:
```
get-vm cr -ErrorAction SilentlyContinue -ErrorVariable getVmErrors
$?
```

would return `False`

```
get-vm crc -ErrorAction SilentlyContinue -ErrorVariable getVmErrors
$?
```

would return `True`

Reply:
Thanks will update it!

Reply:
done and it works perfectly. Thanks for this magic trick!